exonum-java-binding/testkit/src/main/java/com/exonum/binding/testkit/TestKit.java

```diff
@@ -98,6 +99,8 @@
   private static final short TIME_SERVICE_ID = 4;
 
   private final Map<Short, Service> services = new HashMap<>();
+  @VisibleForTesting
+  final List<Cleaner> snapshotCleaners = new ArrayList<>();
```

Comment:
Why is there a List and not a single Cleaner? If the snapshots are destroyed at the same time, a single Cleaner will work (and will be more reliable in case of errors).